src/overlays/actors/ovl_Bg_Jya_Bigmirror/z_bg_jya_bigmirror.c

```diff
@@ -34,18 +35,13 @@ typedef struct {
     /* 0x0C */ s16 params;
     /* 0x0E */ s16 solvedRotY;
     /* 0x10 */ s16 initRotY;
-} BigMirrorDataEntry; // size = 0x14
+} BigMirrorDataEntry; // size = 0x12
```

Comment:
0x14 is the correct size here, since the struct includes 32 bit members (with the Vec3f)
```suggestion
} BigMirrorDataEntry; // size = 0x14
```